ecologits/model_repository.py

```diff
@@ -92,6 +93,8 @@ def from_json(cls, filepath: Optional[str] = None) -> "ModelRepository":
         with open(filepath) as fd:
             data = json.load(fd)
             mf = Models.model_validate(data)
+        if mf.models is None:
+            raise ValueError("Cannot initialize and empty model repository.")
```

Comment:
nitpick: "Cannot initialize and empty model repository." --> "Cannot initialize on an empty model repository." ?